### PR TITLE
Add "go back" possibility to cluster login quick picks #3357

### PR DESCRIPTION
**Pull request by vrubezhny**
Fixes: #3357

### CODE REVIEW

**Comment by vrubezhny** [comment]
How `Back` buttons work for navigation through the Cluster Login Quick Picks:

screencast-cluster-login-back-buttons

**Review by datho7561 — DISMISSED** [review]
Seems to work well. Found some dead code

File: src/openshift/cluster.ts [hunk 1]
```diff
@@ -278,89 +302,163 @@ export class Cluster extends OpenShiftItem {
         return versions;
     }
 
+    showLoginActionQuickPick() {
```
[the remark this answers is not in the document]
Comment (vrubezhny) [reply src/openshift/cluster.ts:305]: yes. Sorry, forgot to remove that function.

**Review by datho7561 — APPROVED** [review]
Looks good. Thanks, Victor!

**Pull request merged**